tests/test_api_errors.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from flask.testing import FlaskClient
+from flask import Response
+from flask_api import status
+
+from .fixtures.client import client
+
+
```

Comment:
Please add any additional error-related tests that you think might be useful!

Reply:
Ideas...

- Non-API 404 returning HTML (probably belongs in a different file)
- Somehow trigger a 500 status (do you know of a way?)
- Errors built in to various other API calls (user not found, mod not published, etc.)

Reply:
Hmm, those tests worked fine locally...

Reply:
Aha, we need to mock up some config.

```
        protocol = _cfg("protocol")
        domain = _cfg("domain")
        if not protocol or not domain:
>           abort(404)
```